src/wechaty_puppet_itchat/puppet.py

```diff
@@ -183,6 +176,7 @@ def __init__(self, options: PuppetOptions, name: str = 'puppet_itchat'):
         self.puppet_options = None
         self.puppet = self
         self.itchat = itchat
+        self.loop = asyncio.get_event_loop()
```

Comment:
Why do you add this loop at here? there will be only one event loop, which can be accessed through `asyncio.get_event_loop()` everywhere, in the bot.

Reply:
Since I am pushing the changes based on PR #32 , I forgot to delete this part of the code...